TF-37107 Add TFE deprecation notice for Redis 6.2 and 6.4 (#2334)
hashicorp/web-unified-docs: content/terraform-enterprise/2.0.x/docs/enterprise/releases/2.0.x/index.mdx

## Deprecations
1. `tfectl` database commands, which were deprecated in 1.2.0, are no longer supported and have been removed.
1. Underlying processes are no longer managed by supervisord. As a result, `supervisortctl` commands have been removed and are no longer supported.

> 1. Redis 6.2 and 6.4 have reached vendor end-of-life and will be officially deprecated in 2.1.0.

Reviewer: ```suggestion
1. Redis 6.2 and 6.4 have reached vendor end-of-life and are deprecated,  Redis 6 support will be removed in 2.1.0
```

Reviewer 2: 🔎 "are deprecated". Plural subject (Redis 6.2 and 6.4). 🤓 . But yes, we're deprecating it now, and removing it in 2.1.0.

> 1. Redis 6.2 and 6.4 have reached vendor end-of-life and are deprecated,  Redis 6 support will be removed in 2.1.0

Reviewer: ```suggestion
1. Redis 6.2 and 6.4 have reached vendor end-of-life and are deprecated,  Redis 6.x support will be removed in 2.1.0
```

> 1. Redis 6.2 and 6.4 have reached vendor end-of-life and are deprecated,  Redis 6.x support will be removed in 2.1.0

Reviewer 3: ```suggestion
1. Redis 6.2 and 6.4 have reached vendor end-of-life and are deprecated. Redis 6.x support will be removed in 2.1.0.
```
